frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -366,8 +368,11 @@ function* maybeLoadIsosurface(
   }`;
   const tracingStoreUrl = `${tracingStoreHost}/tracings/volume/${layer.name}`;
   const volumeTracing = yield* select((state) => getActiveSegmentationTracing(state));
-  // Fetch from datastore if no volumetracing exists or if the tracing has a fallback layer.
-  const useDataStore = volumeTracing == null || volumeTracing.fallbackLayer != null;
```

Comment:
@fm3 This condition was adapted so that even for volume tracings with a fallback layer, the isosurface is requested from the tracing store (to make isosurfaces for editable mappings work). Does this break functionality for tracings without an editable mapping or is the isosurface request proxied to the data store in that case?
If this breaks stuff, I need to adapt the condition and check whether an editable mapping is active as well.

Reply:
I’d say if your request an isosurface from a normal volume annotation with fallback layer *from the tracingstore* it will not do proxying but instead run marching cubes only on the modified buckets. if there are no modified buckets, there will be no isosurfaces. So yes, I’d say this breaks this.
In the long run, I guess we will want this proxying to happen in the backend also in this case, but maybe not as part of this PR

Reply:
@fm3 Is this exposed somehow (whether there are brush strokes in a volume annotation)? In any case, I'd rather not call a route to get this information before each isosurface request. One option could be to request this info once during annotation loading and then keep track of volume modifications in the frontend :shrug: Alternatively, the backend could take care of the proxying, as discussed.
For now, the simplest solution is probably to restore this condition as it was before. Then, adjust the condition to request from the tracing store if and editable mapping is active.

Reply:
> What downsides do you see for your last suggestion?

Good question - For the purposes of this PR, I don't see any actually :)

Reply:
One upside of an exposed route would be that the proofreading tool could be disabled from the beginning if an annotation with volume modifications is opened. However, not that important for the first iteration.